test/e2e/storage/vsphere/vsphere_volume_node_poweroff.go

```diff
@@ -147,20 +147,18 @@ var _ = utils.SIGDescribe("Node Poweroff [Feature:vsphere] [Slow] [Disruptive]",
 // Wait until the pod failed over to a different node, or time out after 3 minutes
 func waitForPodToFailover(client clientset.Interface, deployment *appsv1.Deployment, oldNode string) (string, error) {
 	var (
-		err      error
-		newNode  string
 		timeout  = 3 * time.Minute
 		pollTime = 10 * time.Second
 	)
 
-	err = wait.Poll(pollTime, timeout, func() (bool, error) {
-		newNode, err = getNodeForDeployment(client, deployment)
+	err := wait.Poll(pollTime, timeout, func() (bool, error) {
+		currentNode, err := getNodeForDeployment(client, deployment)
 		if err != nil {
 			return true, err
 		}
 
-		if newNode != oldNode {
-			framework.Logf("The pod has been failed over from %q to %q", oldNode, newNode)
+		if currentNode != oldNode {
+			framework.Logf("The pod has been failed over from %q to %q", oldNode, currentNode)
```

Comment:
is this log necessary? It should be fairly clear this happened from a successful exit of the function `waitForPodToFailover`

Reply:
it didn't seem like it was hurting anyone. Probably this level of logging belongs in the caller. How much do you care about cleaning this one up?

Reply:
not enough